### PR TITLE
Add Help steer dataset

### PR DESCRIPTION
**Pull request by vumichien**
Hi @shayne-longpre

This PR adds the Help steer Dataset (with the downloader, preparer, and mapper). All tests passed successfully.

Tell me if you need to change anything.

### CODE REVIEW

File: data_summaries/HelpSteer.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            "Open Question Answering",
+            "Closed Question Answering"
+        ],
+        "Text Sources": [
+            "scale.com",
+            "human"
+        ],
+        "Model Generated": [],
+        "Format": [
+            "Chain-of-Thought"
```
Comment (shayne-longpre) [inline data_summaries/HelpSteer.json:32]: zero-shot?

File: data_summaries/HelpSteer.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [""],
```
Comment (shayne-longpre) [inline data_summaries/HelpSteer.json:35]: empty list

File: data_summaries/HelpSteer.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        "Creators": [
+            "Nvidia"
+        ],
+        "Licenses": [
+            {
+                "License": "CC BY-NC 4.0",
```
Comment (shayne-longpre) [inline data_summaries/HelpSteer.json:41]: Looks like its actually CC BY 4.0? Not NC.

File: data_summaries/No Robots.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
+{
+    "no-robots": {
```
Comment (shayne-longpre) [inline data_summaries/No Robots.json:2]: Disentangle this PR from no-robots and privacy-200

**Review by shayne-longpre — APPROVED** [review]
Approved once fixes made and tests pass without any warnings! Thank you!

File: data_summaries/No Robots.json (hunk 4 above)
Comment (vumichien) [reply data_summaries/No Robots.json]: Fix this PR disentangle from privacy-200 and no-robot

File: data_summaries/HelpSteer.json (hunk 1 above)
Comment (vumichien) [reply data_summaries/HelpSteer.json]: It is revised as a comment.

File: data_summaries/HelpSteer.json (hunk 2 above)
Comment (vumichien) [reply data_summaries/HelpSteer.json]: It is revised as a comment.

**Comment by shayne-longpre** [comment]
@vumichien this looks great! Resolve conflicts with main then it should be ready to merge after it passes all tests

**Comment by vumichien** [comment]
@shayne-longpre All conflicts were resolved

**Pull request merged**